forwarder/forwarder/forwarderobject.py

```diff
@@ -58,8 +60,8 @@ class Forwarder(Process):
     out from some DB ?
     """
 
-    def __init__(self, task_q, executor, endpoint_id,
-                 heartbeat_threshold=60, endpoint_addr=None,
+    def __init__(self, task_q, task_status_q, executor, endpoint_id,
+                 heartbeat_threshold=30, endpoint_addr=None,
```

Comment:
I would recommend setting the threshold back to 60.

Reply:
Changed to 60!